openlibrary/plugins/upstream/addbook.py

```diff
@@ -406,6 +406,7 @@ def save(self, formdata):
                 self.delete(self.work.key, comment=comment)
             return
 
+        just_editing_work = edition_data is None
```

Comment:
This confused me at first, but I see that it is due to the combined work and edition edit page. I'm not sure what value combining them brings as it just seems to make it harder to track what is being edited. Not relevant for this accepting this PR, but I'll bring up the discussion to see if there is support from making the distinction between work and edition edits clearer.